doc_admin/AdminManual.adoc

```diff
@@ -2012,8 +2045,39 @@ This file defines the parameters for sending SMS using the Skebby HTTP API.
 * *key*: the API key (if using a  purchased sms) from Textbelt provider (one can use "textbelt" to send 1 free sms per day)
 * *ribbon.base-url*: The URL of the Textbelt HTTP API.
 
+[#telemetry-properties]
+=== 3.7 telemetry.properties
+
+The Telemetry configuration is set in the telemetry.properties_ file:
+
+[source,properties]
+----
+# number of second to check if it should send a message
+telemetry.daemon.thread.loop.seconds=14400
+
+# enabled remote geo ip lookup service
+# allowed values:
+# - geoiplookup-remote-service
+# - ipapi-remote-service
+telemetry.enabled.geo.ip.lookup.service=geoiplookup-remote-service
+# endpoints remote services
+geoiplookup-remote-service.ribbon.base-url=https://json.geoiplookup.io
+ipapi-remote-service.ribbon.base-url=http://ip-api.com/json 
+----
+
+This file is used only if the TELEMETRYENABLED flag is true (see <<telemetryenabled,TELEMETRYENABLED>> in this document).
+
+The file has the following configurable parameters:
+
+* *telemetry.daemon.thread.loop.seconds*: number of seconds the Telemetry module should try to send again the telemetry data; if the telemetry data have been sent in the same day it will not send again.
```

Comment:
```suggestion
* *telemetry.daemon.thread.loop.seconds*: number of seconds the Telemetry module should try again to send the telemetry data; if the telemetry data have been sent in the same day it will not send again.
```